ref.html

```diff
@@ -156,11 +156,6 @@
       switched companies during edition) or to clarify that an editor is no longer active even if her
       name is still attached to the document out of respect for her past contributions.
     </dd>
-    <dt>w3cid</dt>
```

Comment:
Damn, sorry, I may have been too quick in merging this. Did you really intend to delete this?